src/crimson/os/seastore/onode_manager/staged-fltree/tree_utils.h

```diff
@@ -320,7 +326,7 @@ class TreeBuilder {
   }
 
   KVPool& kvs;
-  Btree tree;
+  std::optional<Btree> tree;
```

Comment:
why?

Reply:
Because the entire tree needs to be reloaded during replay/remount test. While at runtime, the tree doesn't need to be reloaded.